docs/csharp/tour-of-csharp/enums.md

```diff
@@ -0,0 +1,82 @@
+---
+title: C# Enums | A tour of the C# language
+description: Learn about enums, discrete named constants in C#
+keywords: .NET, csharp 
+author: BillWagner
+manager: wpickett
+ms.date: 08/10/2016
+ms.topic: article
+ms.prod: visual-studio-dev-14
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: 7faba1cc-6ea9-4a19-adb9-0335e4b132e5
+---
+	
+# Enums
+
+An ***enum type*** is a distinct value type with a set of named constants. The following example declares and uses an enum type named `Color` with three constant values, `Red`, `Green`, and `Blue`.
```

Comment:
I feel like the introduction of the example is kind of abrupt. You have just a short summary and immediately comes the following example.